### DISCUSSION TITLE
blog/frontend/vue/[Vue.js]%20Vue.js%20%EB%8F%84%EC%9E%85%EA%B8%B0

### DISCUSSION THREAD

**Discussion by giscus** [swj9707/swj_techblog#3] [Reply]
# blog/frontend/vue/[Vue.js]%20Vue.js%20%EB%8F%84%EC%9E%85%EA%B8%B0

백엔드 개발자의 Vue.js 사용 후기, 장단점 등

https://swj-techblog.vercel.app/blog/frontend/vue/[Vue.js]%20Vue.js%20%EB%8F%84%EC%9E%85%EA%B8%B0

**Comment by shghdrl1234** [comment]
<h1>멋집니다!👍

<h4> 종종 들려서 우진님 경험들을 흡수해 가겠습니다. ㅎㅎ

**Reply by oxspan** [reply]
감사합니다 ㅋㅋㅋ 자주 들려주세요
